components/kyma-environment-broker/common/orchestration/dto.go

```diff
@@ -8,13 +8,18 @@ import (
 
 // Parameters hold the attributes of orchestration create (upgrade) requests.
 type Parameters struct {
-	Targets  TargetSpec   `json:"targets"`
-	Strategy StrategySpec `json:"strategy,omitempty"`
-	DryRun   bool         `json:"dryRun,omitempty"`
+	Targets    TargetSpec           `json:"targets"`
+	Strategy   StrategySpec         `json:"strategy,omitempty"`
+	DryRun     bool                 `json:"dryRun,omitempty"`
+	Kubernetes KubernetesParameters `json:""`
 	// upgrade kyma specific parameters
 	Kyma KymaParameters `json:""`
 }
 
+type KubernetesParameters struct {
+	Version string `json:"kubernetesVersion,omitempty"`
```

Comment:
Adding all the fields from `ClusterDetailResponse`
```suggestion
	KubernetesVersion   string `json:"kubernetesVersion"`
	MachineImage        string `json:"machineImage"`
	MachineImageVersion string `json:"machineImageVersion"`
```